exporter/dynatraceexporter/README.md

```diff
@@ -256,3 +253,24 @@ Default: `5000`
 ### tags (Deprecated, Optional)
 
 **Deprecated: Please use [default_dimensions](#default_dimensions-optional) instead**
+
+# Limitations
```

Comment:
Limitations alone sounds a bit pejorative.. in reality, it's limitation when using cumulatives. Maybe this is a bit better? 
`Limitations when using Cumulatives values`. Or `Particularities when exporting cumulative values`? Naming is so hard..

Reply:
Yep that is a difficult one. But I think it is a necessary eye-catcher for users to see that exporting cumulatives can lead to issues. I feel like using too soft wording could make readers glance over it.

Reply:
Considerations sounds good. Maybe even `Considerations with cumulative values`?

Reply:
Considerations it is, thanks. :)